components/Workouts/WorkoutCard/index.jsx

```diff
@@ -0,0 +1,64 @@
+import Heading from "@/components/Atoms/Text/Heading";
+import HeadingTiny from "@/components/Atoms/Text/HeadingTiny";
+import styled from "styled-components";
+
+export default function WorkoutCard({ workout, exercises }) {
+  const usedMuscle = [];
+  exercises.map((exercise) => usedMuscle.push(...exercise.muscleGroups));
+  const uniqueMuscles = [...new Set(usedMuscle)];
+
+  const muscleCount = {};
+  for (const muscle of usedMuscle) {
+    muscleCount[muscle] = muscleCount[muscle] ? muscleCount[muscle] + 1 : 1;
+  }
+
+  return (
+    <StyledWorkoutCard>
+      <h3>{workout.name}</h3>
+      <>
+        <Heading>Exercises</Heading>
+        {exercises.map((exercise, index) => {
+          const exerciseDataInWorkout = workout.exercises.find(
+            (workoutExercise) => workoutExercise.exerciseId === exercise._id
+          );
+          return (
+            <StyledParagraph key={index}>
```

Comment:
```suggestion
            <StyledParagraph key={exercise._id}>
```
Use `exercise._id` as a key here.